pages/api/explore/people.js

```diff
@@ -0,0 +1,15 @@
+import suggestionsData from '../../../server/suggestions-data.json';
+
+export default function handler(req, res) {
+  const { count } = req.query;
+  if (count) {
+    const SlicedSuggestionsData = suggestionsData.slice(0, count);
+    if (SlicedSuggestionsData) {
+      res.status(200).json(SlicedSuggestionsData);
+    } else {
+      res.status(404).send('Not found.');
+    }
+  } else {
+    res.status(200).json(suggestionsData);
+  }
+}
```

Comment:
Is there a reason this file is called `explore/people.js`? Why not just `suggestions.js`. The API shouldn't need to know where it will be used.

Reply:
Nah, ill fix it.